data/src/main/java/cmm/apps/esmorga/data/user/model/UserDataModel.kt

```diff
@@ -0,0 +1,7 @@
+package cmm.apps.esmorga.data.user.model
+
+data class UserDataModel(
+    val nameData: String,
```

Comment:
in Event the naming convention is "dataXxx" and not "xxxData". Naming be aligned.